lab_2_tokenize_by_bpe/target_score.txt

```diff
@@ -1 +1 @@
-0
+6
```

Comment:
I was secretly hoping you'll aim for 10 :thought_balloon:

Reply:
Well let's just say if the night before hand-in I'll happen to find previously unseen half of the lab I won't be too mad. But it's up to you!

Reply:
I will try to do steps for 10